data/json/items/melee/swords_and_blades.json

```diff
@@ -2574,7 +2577,7 @@
     "price_postapoc": 6000,
     "material": [ "steel" ],
     "flags": [ "DURABLE_MELEE", "SHEATH_SWORD", "ALWAYS_TWOHAND" ],
-    "weapon_category": [ "KNIVES" ],
+    "weapon_category": [ "SHORT_SWORDS", "KNIVES" ],
```

Comment:
Machetes aren't knives, so these shouldn't be either

Reply:
the thing is their shape suggests both a knife and a short sword, so idk

Reply:
but yeah every source refers to them as swords